src/sentry/ingest/ingest_consumer.py

```diff
@@ -64,6 +64,9 @@ def flush_batch(self, batch):
                     other_messages.append((process_userreport, message))
                 else:
                     raise ValueError("Unknown message type: {}".format(message_type))
+                metrics.incr(
+                    "ingest_consumer.flush.messages_seen", tags={"message_type": message_type}
```

Comment:
Isn't this basically the same metric as the one in BatchingKafkaConsumer?

Reply:
Yes, but it's of a different "type", and I just want to confirm that those are consistent.